### DISCUSSION TITLE
roles and permissions snapshot

### DISCUSSION THREAD

**Discussion by terion-name** [directus/directus#10496] [Feature Requests]
snapshotting of schema was added recently (https://github.com/directus/directus/pull/7939) and this is great, but in many cases role and permissions logic can be complex and is a part of predefined application business logic (at minimum it directly affects graphql api schema). 
So it would be very useful to have a possibility to snapshot and apply not only datamodel, but roles and permissions also

**Comment by azrikahar** [comment]
This was recently discussed over at #9542 👍 There isn't an out-of-the-box direct solution now, but in the meantime you can get the list of permissions via the `/permissions` API then make POST request(s) to create/apply them, as described here: https://github.com/directus/directus/discussions/9542#discussioncomment-1596860

Let's consolidate this discussion over there going forward 😊

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
